packages/actor-query-operation-sparql-endpoint/package.json

```diff
@@ -41,7 +41,7 @@
     "@comunica/mediatortype-httprequests": "^2.3.0",
     "@comunica/types": "^2.3.0",
     "@rdfjs/types": "*",
-    "asynciterator": "^3.4.2",
+    "asynciterator": "^3.7.0",
```

Comment:
Could you also update this dep in all our other packages? Just so we avoid multiple different versions of asynciterator. (there are two of them now in the `yarn.lock` file)

Reply:
Related to this, we may also want to depend on the latest asyncjoin version, to avoid dep issues downstream.